service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/HearingBookingDetailsControllerTest.java

```diff
@@ -70,28 +78,76 @@ void shouldNotReturnAnErrorWhenHearingDateIsSetInDistantFuture() throws Exceptio
         LocalDateTime distantFuture = LocalDateTime.now().plusYears(1000);
 
         AboutToStartOrSubmitCallbackResponse callbackResponse = makeRequest(createHearing(
-            distantFuture, distantFuture.plusDays(1)));
+            distantFuture, distantFuture.plusDays(1)), EMPTY_BOOKING);
+
+        assertThat(callbackResponse.getErrors()).doesNotContain(ERROR_MESSAGE);
+    }
+
+    @Test
+    void shouldOnlyValidateNewBookingsWhenBookingAlreadyExists() throws Exception {
+        LocalDateTime distantFuture = LocalDateTime.now().plusYears(1000);
+        LocalDateTime distantPast = LocalDateTime.now().minusYears(10000);
+
+        AboutToStartOrSubmitCallbackResponse callbackResponse = makeRequest(createHearing(
+            distantFuture, distantFuture.plusDays(1)), createHearing(distantPast, distantPast.plusDays(1)));
 
         assertThat(callbackResponse.getErrors()).doesNotContain(ERROR_MESSAGE);
     }
 
+    @Test
+    void shouldErrorWhenExistingBookingIsUpdatedToPastDate() {
+        LocalDateTime futureDate = LocalDateTime.now().plusDays(5);
+        UUID hearingId = randomUUID();
+
+        CallbackRequest request = callbackRequestWithEditedBooking(futureDate, hearingId);
```

Comment:
Not ideal test, as we don't see on a glance what is previous date, but it's minor for me as the method is directly below so easy to find out.